contracts/Vault.vy

```diff
@@ -1412,7 +1419,6 @@ def _reportLoss(strategy: address, _loss: uint256):
     debtRatio: uint256 = self.strategies[strategy].debtRatio
     self.strategies[strategy].debtRatio -= min(loss * MAX_BPS / self._totalAssets(), debtRatio)
 
-
```

Comment:
Maintain 2 spaces between each function please

Reply:
fixed in latest